### PR TITLE
doc-390

### PR DESCRIPTION
**Pull request by resmaa**
**Minimum limits added for** 
1. Sandbox clusters
2. Cluster Groups
3. Resource Quota

Specified for cluster group, host clusters need to be launched under Tenant Scope.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/961#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (amarjeet27) [inline content/docs/04.5-devx/07-sandbox-clusters.md:43]: The minimum for launching sandbox clusters is 2.5 CPU, 2.5 GiB memory, 0 GiB storage.
Also see https://spectrocloud.atlassian.net/browse/PPD-618

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/03-cluster-groups.md:30]:
```suggestion
A Cluster Group is an aggregation of host clusters deployed through the Palette Tenant Admin console. Palette virtual clusters can be launched into host cluster groups. The cluster groups are created under two scopes:
```

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/03-cluster-groups.md:93]:
```suggestion
1. Select the Palette Cluster Group to be managed and click **Settings** to access different cluster management activities. 
```

File: content/docs/04.5-devx/04-resource-quota.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/04-resource-quota.md:35]:
```suggestion
3. Set the desired *User Quotas* values.  The user quotas have the following default values:
```
Because we are not referring to the UI element but rather the label, we can use italics the first time.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/961#pullrequestreview-1167155133" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/961#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:47]:
```suggestion
   * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
```

**Pull request merged**
